serve_css.py

```diff
@@ -0,0 +1,8 @@
+@app.get('/question/<number>')
+def question(number):
+    statement = questions[number].statement
+    return template('index.html', question_number=number, question=statement)
+    return template('index.css', question_number=number, question=statement)
```

Comment:
you can only return once from a function. The other returns will never be executed.